v3/newrelic/config.go

```diff
@@ -48,6 +48,11 @@ type Config struct {
 	// https://docs.newrelic.com/docs/using-new-relic/user-interface-functions/organize-your-data/labels-categories-organize-apps-monitors
 	Labels map[string]string
 
+	// CustomAttributes A hash with key/value pairs to add as custom attributes to all log events forwarded to New Relic.
+	//
+	// https://docs.newrelic.com/docs/TODO
+	CustomAttributes map[string]string
```

Comment:
I would recommend moving this so all of the options for the feature are located together (the enable boolean and this map), either as a pair of values in `CustomInsightsEvents` like
```
CustomInsightsEvents struct {
   CustomAttributesEnabled bool
   CustomAttributesValues map[string]string
}
```
or as a sub-struct like
```
CustomInsightsEvents struct {
   CustomAttributes struct {
      Enabled bool {
      Values map[string]string
   }
}
```
(or whatever is most appropriate for naming instead of `Values` there), so they're not separated from each other in the `Config` struct.